packages/core-js/utils/contracts.js

```diff
@@ -22,7 +22,8 @@ async function getSelectors(contractAbi) {
   );
 
   return contract.interface.fragments.reduce((selectors, fragment) => {
-    if (fragment.type === 'function') {
+    // Only add methods, and filter out the ones added during instrumentation by solidity-coverage
+    if (fragment.type === 'function' && !fragment.name.startsWith('c_')) {
```

Comment:
The filtering out of `c_*` methods is necessary so we don't add `solidity-coverage` instrumentation methods to the `Router.sol` file.

More description & follow up here: https://github.com/Synthetixio/synthetix-v3/issues/477